marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mappingWithCustomHeader.cy.tsx

```diff
@@ -96,14 +91,7 @@ describe("Create and verify load steps, map step and flows with a custom header"
   it("Edit Map step", () => {
     cy.visit("/tiles/curate");
     cy.waitForAsyncRequest();
-
-    cy.log("**Open Order to see steps**");
-    curatePage.getEntityTypePanel("Order").then(($ele) => {
-      if ($ele.hasClass("accordion-button collapsed")) {
-        cy.log("**Toggling Entity because it was closed.**");
-        curatePage.toggleEntityTypeId("Order");
-      }
-    });
+    curatePage.toggleEntityTypeId("Order");
```

Comment:
We should use the expandAccordion method here.

Reply:
Basically, if you navigate directly using the link, never keeps open. When you use the toolbar, you need to know the previous step, or use the expandAccordion. In this case, it will not keep open, because we are navigating directly to the link, so it is not needed. 

I am going to create a task to standardize how to do it, but basically is not needed to change this.